lib/ProductOpener/ForestFootprint.pm

```diff
@@ -321,14 +318,8 @@ add_footprint($product_ref, $ingredient_ref, $footprints_ref, $ingredients_categ
 					
 =cut
 
-sub add_footprint ($$$$$) {
+sub add_footprint ($product_ref, $ingredient_ref, $ingredients_category_ref, $footprint_ref, $footprints_ref) {
```

Comment:
```suggestion
sub add_footprint ($product_ref, $ingredient_ref, $footprints_ref, $ingredients_category_ref, $footprint_ref) {
```
Careful with the order. It needs to match the old one exactly.

Reply:
Sorry! I didn't know that the order mattered